jenkins-x.yml

```diff
@@ -7,6 +7,17 @@ pipelineConfig:
           image: gcr.io/jenkinsxio/builder-go
         stages:
         - name: pr-checks
+          options:
+            containerOptions:
+              volumeMounts:
+              - mountPath: /builder/home/.jx/localSecrets/currentCluster
```

Comment:
What I'll happen when vault is configured as a secrets source. How I understand this volume is always mounted. It probably shouldn't be mounted when vault is enabled.

Reply:
The secret will only be created when we boot with local secrets and the vault is marked as optional, so it will not be mounted.

Reply:
cool! I missed the `optional`.